resources/components/ImageCard.vue

```diff
@@ -43,13 +48,15 @@
 </template>
 
 <script>
-var Button = require( './base/Button.vue' );
+var Button = require( './base/Button.vue' ),
+	Suggestion = require( './base/Suggestion.vue' );
 
 module.exports = {
 	name: 'ImageCard',
 
 	components: {
-		'base-button': Button
+		'base-button': Button,
```

Comment:
Maybe we should start referring to this as `mw-button` in the templates since that's how we're naming the classes.